CRM/Financial/BAO/FinancialItem.php

```diff
@@ -47,7 +47,7 @@ public function __construct() {
    * @param array $defaults
    *   (reference ) an assoc array to hold the flattened values.
    *
-   * @return CRM_Contribute_BAO_FinancialItem
+   * @return CRM_Financial_BAO_FinancialItem
```

Comment:
@eileenmcnaughton i think this one liner could be spun off into that other NFC change PR

Reply:
done

Reply:
(actually I think it's in the 'add test' per too)